src/plugins/irc-events/link.js

```diff
@@ -159,7 +159,14 @@ function fetch(uri, cb) {
 	var limit = Helper.config.prefetchMaxImageSize * 1024;
 	req
 		.on("response", function(res) {
-			if (!(/^image\/.+/.test(res.headers["content-type"]))) {
+			if ((/^image\/.+/.test(res.headers["content-type"]))) {
+				// response is an image
+				// if Content-Length header reports a size exceeding the prefetch limit, abort fetch
+				const contentLength = parseInt(res.headers["content-length"], 10) || NaN;
+				if (contentLength && contentLength > limit) {
+					res.req.abort();
```

Comment:
This can be `res.abort`.

Reply:
OK, I was copying the code in the pipe handler below because I couldn't find a reference to `abort()` in the docs for `request`. For future reference, where is it documented that you can call `abort()` directly on the `response` passed to this event handler?